changelog.md

```diff
@@ -4,6 +4,10 @@ Changes to the TinyMCE documentation are documented in this file.
 
 The format is loosely based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+### Unreleased
+
+- DOC-2026: AI Assistant Premium plugin documentation.
```

Comment:
@brian-forte do we usually include keyboard shortcut information. Reference: https://github.com/tinymce/ai/blob/main/src/main/ts/api/Keyboard.ts

@MitchCTiny FYI

Reply:
We’ve done it once in `advcode.adoc`:

`include::partial$misc/advcode-shortcuts.adoc[]`

There is also `/pages/keyboard-shortcuts.adoc` (presents as [keyboard-shortcuts](https://tiny.cloud/docs/tinymce/6/keyboard-shortcuts/) in the docs).

There is no automated or templated infrastructure in place to do this with all new plugins.